htdocs/core/modules/modCategorie.class.php

```diff
@@ -821,14 +821,15 @@ protected function exportTagLinks(int $r, string $categcode, string $class, stri
 		$keyforselect = $class;
 		$keyforelement = $class;
 		$keyforaliasextra = 'extra';
+		$class_table = (new $class($db))->table_element;
```

Comment:
I cant find wherecis the include that guarantee the new will not fails ?

Reply:
Either it is up to the caller,
or there a way load all classes,
or there is a way to match "CommandeFournisseur" to "htdocs/fourn/class/fournisseur.commande.class.php"
or there is a way to match "CommandeFournisseur" to "commande_fournisseur" for the table name
or files needs to be "renamed" (while keeping backward compatibility)
and/or there should be a function "dol_include_class" that would include the proper file.


getElementProperties does not help.

So, the class file needs to be included by the caller which is luckily only found in the same file.

The code is updated accordingly.